crm_claim_rma/models/account_invoice.py

```diff
@@ -13,47 +13,53 @@ class AccountInvoice(models.Model):
 
     claim_id = fields.Many2one('crm.claim', string='Claim')
 
+    @api.model
     def _refund_cleanup_lines(self, lines):
-        """ Override when from claim to update the quantity and link to the
-        claim line.
+        """ Override when from claim to update the quantity, link to the
+        claim line and return empty list for tax_line_ids.
         """
+        claim_lines_context = self._context['claim_line_ids']
```

Comment:
1. always try to use public methods/variables when they are available
```suggestion
        claim_lines_context = self.env.context['claim_line_ids']
```
2. Is `claim_line_ids` guaranteed to be in the context. I assume it is not based on this [code](https://github.com/versada/rma/blob/e0b02f63de1991ad6bbb0331752ffb76810eb2a6/crm_claim_rma/models/account_invoice.py#L22-L23) so you need to account for that